### PR TITLE
feat(starfish): Add per second unit to y axes of throughput charts

### PR DESCRIPTION
**Pull request by 0Calories**
Adds `/s` for the throughput charts throughout the app

image

### CODE REVIEW

File: static/app/utils/discover/charts.tsx [hunk 1]
```diff
@@ -92,6 +92,8 @@ export function axisLabelFormatterUsingAggregateOutputType(
       return axisDuration(value, durationUnit);
     case 'size':
       return formatBytesBase2(value, 0);
+    case 'rate':
+      return `${value}/s`;
```
Comment (gggritso) [inline static/app/utils/discover/charts.tsx:96]: Oohh, risky! I think this should accept a `rateUnit`, there have been some discussions that per second might not make sense for people with lower throughput. Is it a lot of work to do that?

**Review by gggritso — COMMENTED** [review]
LGTM, but I think having a unitless `rate` always use `/s` might be risky!

File: static/app/utils/discover/charts.tsx (hunk 1 above)
Comment (0Calories) [reply static/app/utils/discover/charts.tsx:96]: You're right, I thought about this but thought it wasn't necessary. I can certainly look into making it dynamic based on whether it's higher or lower throughput, but maybe we should wait for confirmation on that before I work on it?

Comment (gggritso) [reply static/app/utils/discover/charts.tsx:96]: Take a look, IMO! Fingers crossed it's easy, and if yes, it'll be so worth it. If not, then no worries. If it was just Starfish code I wouldn't even ask, but this is common Discover stuff, and it deserves to be solid

Comment (0Calories) [reply static/app/utils/discover/charts.tsx:96]: Fair point, lemme go do that then

**Comment by 0Calories** [comment]
@gggritso I added a function to dynamically update the rate unit based on some hardcoded values, do you think this is sufficient or were you thinking of something else?

**Comment by gggritso** [comment]
@0Calories oh, that's actually more involved than I expected! When I was looking at `charts.tsx`, I saw this:

https://github.com/getsentry/sentry/blob/master/static/app/utils/discover/charts.tsx#L91-L92

When the `type` is `"duration"`, it also has an optional `durationUnit` prop. I was thinking of exactly the same thing for `rate`. You don't need to dynamically calculate it, just to _allow_ passing a different `rateUnit`, that's all! It can be `seconds` by default, even

**Comment by 0Calories** [comment]
Oh gotcha, I saw this as well but I thought we wanted something dynamic haha. I'll change this up

**Comment by 0Calories** [comment]
How's this @gggritso ?

**Review by gggritso — APPROVED** [review]
Beautiful!

**Pull request merged**